pvlib/location.py

```diff
@@ -99,8 +99,9 @@ def from_tmy(cls, tmy_metadata, tmy_data=None, **kwargs):
 
         Returns
         -------
-        Location object (or the child class of Location that you
-        called this method from).
+        new_object
+            Location object (or the child class of Location that you
+            called this method from).
```

Comment:
I'm guessing I wrote this a long time ago but I don't like it. Let's just make it

```
Returns
-------
Location
```